cli-commands/compiler/etherlime-resolver/index.js

```diff
@@ -36,33 +36,33 @@ Resolver.prototype.resolve = function (import_path, imported_from, callback) {
   whilst(function () {
     return !resolved_body && current_index < self.sources.length - 1;
   }, function (next) {
-    current_index += 1;
-    current_source = self.sources[current_index];
+      current_index += 1;
+      current_source = self.sources[current_index];
 
-    current_source.resolve(import_path, imported_from, function (error, body, file_path) {
-      if (!error && body) {
-        resolved_body = body;
-        resolved_path = file_path;
+      current_source.resolve(import_path, imported_from, function (error, body, file_path) {
+        if (!error && body) {
+          resolved_body = body;
+          resolved_path = file_path;
+        }
+
+        next(error);
+      });
+    }, function (error) {
+      if (error) {
+        return callback(error);
```

Comment:
I think row 51 is dead code, we never has error, but will be glad if any of you can check it out too. :)